src/Twig/Extension/ArrayExtension.php

```diff
@@ -188,4 +203,28 @@ public function unique(array $arr1, array $arr2)
 
         return $compiled;
     }
+
+    /**
+     * Tests if a given value is an array or not, returns the result.
+     *
+     * @param mixed $value
+     *
+     * @return bool
+     */
+    public function array($value)
```

Comment:
Dare I say it, that would have shown up if you were using PhpStorm and not that other one :stuck_out_tongue_winking_eye: